projects/packages/identity-crisis/changelog/update-backport-packages-9.9

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
This file probably shouldn't be being created. That's why things are wanting to update it to 0.2.2-alpha.

The removal of update-manager_get_connected_users should be enough to cover the updates going on in this PR.

Reply:
Hmm, the tests were failing and looking for a changelog file for that. Should I undo the last commit where I updated the versions? I was trusting the tests 😬